src/main/kotlin/org/wa/rceditor/application/view/fragments/DialogFragment.kt

```diff
@@ -0,0 +1,49 @@
+package org.wa.rceditor.application.view.fragments
+
+import javafx.geometry.Pos
+import javafx.scene.layout.Priority
+import javafx.scene.text.TextAlignment
+import tornadofx.*
+
+class DialogFragment: Fragment() {
```

Comment:
Does TornadoFX not have a built-in alert dialog we could use here?

Reply:
@mbr4477 It does, but I didn't like it. It doesn't wrap the long text

Reply:
Interesting, good to know. Was there not any way to set the isWrapText property on the built-in dialog's message label?

Reply:
@mbr4477 I didn't find the way to customize it